src/backends/pyplot.jl

```diff
@@ -1006,9 +1006,17 @@ function _before_layout_calcs(plt::Plot{PyPlotBackend})
             # create and store the colorbar object (handle) and the axis that it is drawn on.
             # note: the colorbar axis is positioned independently from the subplot axis
             fig = plt.o
-            cbax = fig."add_axes"([0.8,0.1,0.03,0.8], label = string(gensym()))
+
+            #  There is no point specifying colorbar size, hence rand(4) as it will get
+            #  updated at function _update_plot_object(plt::Plot{PyPlotBackend})
+            cbax = fig."add_axes"(rand(4), label = string(gensym()))
```

Comment:
Wouldn't `zeros` or `ones` be a better choice than `rand` then?

Reply:
Right, perhaps I should use zeros(4) as it is cognitively easier than rand(4)